enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -29,63 +29,48 @@
   (when (require-authz?)
     (api/check-superuser)))
 
+(declare execute!*)
+
 (api.macros/defendpoint :post "/table/:table-id"
   "Insert row(s) into the given table."
   [{:keys [table-id]} :- [:map [:table-id ms/PositiveInt]]
    {}
-   {:keys [rows scope]} :- [:map
-                            [:rows [:sequential {:min 1} :map]]
-                            ;; TODO make this non-optional in the future
-                            [:scope {:optional true} ::types/scope.raw]]]
+   {:keys [rows]} :- [:map [:rows [:sequential {:min 1} :map]]]]
   (check-permissions)
-  (let [;; This is a bit unfortunate... we ignore the table-id in the path when called with a custom scope...
-        ;; The solution is to stop accepting custom scope once we migrate the data grid to action/execute
-        scope (or scope {:table-id table-id})]
-    {:created-rows (map :row (:outputs (actions/perform-action! :data-grid.row/create scope rows)))}))
+  (let [scope {:table-id table-id}]
+    {:created-rows (map :row (execute!* "data-grid.row/create" scope {} rows))}))
 
 (api.macros/defendpoint :put "/table/:table-id"
   "Update row(s) within the given table."
   [{:keys [table-id]} :- [:map [:table-id ms/PositiveInt]]
    {}
-   {:keys [rows pks updates scope]}
+   {:keys [rows pks updates]}
    :- [:multi {:dispatch #(cond
                             (:rows %) :mixed-updates
                             (:pks %)  :uniform-updates)}
-       [:mixed-updates [:map
-                        [:rows [:sequential {:min 1} :map]]
-                        ;; TODO make :scope required
```

Comment:
Actually made more sense to remove it, since this API is now only there for tests and 3rd parties to use.